src/layouts/MainLayout.vue

```diff
@@ -73,8 +73,9 @@ q-layout(
     router-view
 
   q-footer(style="height: 200px;").ossph-gradient-bg.q-pb-md.text-center.text-white
-    q-toolbar.q-gutter-sm.q-my-lg
-      q-space
+    q-toolbar(
+      class="soc-med-toolbar"
```

Comment:
Hi, same comment as this one 👉 https://github.com/OSSPhilippines/ossph.org/pull/22#discussion_r1174522096

Reply:
> Hi, same comment as this one 👉 [#22 (comment)](https://github.com/OSSPhilippines/ossph.org/pull/22#discussion_r1174522096)

Have made the changes sir @jofftiquez. As well as in #22

Reply:
Hey thanks @fidellim and please, drop the "sir" 😄

Reply:
hahaha alright! @jofftiquez 🤝